### PR TITLE
feat (migration) : create employee table

### PR DESCRIPTION
**Pull request by mhanifti**
- function up() on create_employee_table.php
- function down() on create_employee_table.php

solve #1

### CODE REVIEW

File: database/migrations/2024_09_28_053328_create_employee_table.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('employee', function (Blueprint $table) {
+            $table->uuid('id');
+            $table->string('name');
+            $table->year('birth_year');
```
Comment (DeanTevin) [inline database/migrations/2024_09_28_053328_create_employee_table.php:17]: add address: after birth year

**Review by DeanTevin — COMMENTED** [review]
add address after birth year

File: database/migrations/2024_09_28_053328_create_employee_table.php (hunk 1 above)
Comment (mhanifti) [reply database/migrations/2024_09_28_053328_create_employee_table.php:17]: nanti malem

Comment (DeanTevin) [reply database/migrations/2024_09_28_053328_create_employee_table.php:17]: ok good shit

**Review by DeanTevin — APPROVED** [review]
Ok, i'll gonna close this. Thank you

**Pull request merged**